takwimu/templates/profile/profile_detail_base.html

```diff
@@ -130,94 +132,109 @@
 
   {{ block.super }}
 
-    <!-- Takwimu & HURUmap  -->
-    <script>
+  <!-- Takwimu & HURUmap  -->
+  <script>
       // I. Takwimu 
       var takwimu = window.takwimu || {};
       takwimu.page = takwimu.page || {};
       takwimu.page.id = 'topic';
       takwimu.url = '{{ HURUMAP.url }}';
       takwimu.countries = {{ countries|jsonify|safe }};
-      takwimu.settings = {
-        navigation: {
-          country_analysis: '{{ settings.takwimu.NavigationSetting.country_analysis|richtext }}',
-          data_by_topic: '{{ settings.takwimu.NavigationSetting.data_by_topic|richtext }}'
-        },
-        support: {
-          hello: '{{ settings.takwimu.SupportSetting.hello|safe }}',
-          support: '{{ settings.takwimu.SupportSetting.support|safe }}'
-        },
-        socialMedia: {
-          facebook: '{{ settings.takwimu.SocialMediaSetting.facebook|safe }}',
-          twitter: '{{ settings.takwimu.SocialMediaSetting.twitter|safe }}',
-          medium: '{{ settings.takwimu.SocialMediaSetting.medium|safe }}',
-          linkedin: '{{ settings.takwimu.SocialMediaSetting.linkedin|safe }}'
-        }
-      };
+      takwimu.summaries = {{ summaries|jsonify|safe }}
```

Comment:
Since HURUmap data is called `profileData`, lets call these something like `profileDataSummaries` to emphasize the relation.